.github/workflows/upgrade_trunk.yaml

```diff
@@ -0,0 +1,19 @@
+name: Nightly
+on:
+  push: 
+    branches: [sxlijin-patch-2]
+  workflow_dispatch: {}
+permissions: read-all
+jobs:
+  trunk_upgrade:
+    name: Upgrade Trunk
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # For trunk to create PRs
+      pull-requests: write # For trunk to create PRs
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      # >>> Install your own deps here (npm install, etc) <<<
```

Comment:
We don't have a `.trunk/setup-ci/action.yaml`, so we'll need to at least npm install here